osu.Game.Rulesets.Catch/UI/SkinnableCatcher.cs

```diff
@@ -26,8 +25,8 @@ public SkinnableCatcher()
             : base(new CatchSkinComponentLookup(CatchSkinComponents.Catcher), _ => new DefaultCatcher())
         {
             Anchor = Anchor.TopCentre;
-            // Sets the origin roughly to the centre of the catcher's plate to allow for correct scaling.
-            OriginPosition = new Vector2(0.5f, 0.06f) * Catcher.BASE_SIZE;
+            Origin = Anchor.TopCentre;
+            CentreComponent = false;
```

Comment:
This had to be turned off since skin implementations can be of different sizes now.